src/atmos_spectral/driver/solo/mixed_layer.F90

```diff
@@ -643,21 +646,42 @@ subroutine mixed_layer (                                               &
 
 !s Surface heat_capacity calculation based on that in MiMA by mj
 
+
 if(do_sc_sst) then !mj sst read from input file
-         ! read at the new time, as that is what we are stepping to
-         call interpolator( sst_interp, Time_next, sst_new, trim(sst_file) )
+     ! read at the new time, as that is what we are stepping to
+     call interpolator( sst_interp, Time_next, sst_new, trim(sst_file) )
 
-         if(specify_sst_over_ocean_only) then
-	     where (.not.land_ice_mask) delta_t_surf = sst_new - t_surf
-             where (.not.land_ice_mask) t_surf = t_surf + delta_t_surf			 
+     if(specify_sst_over_ocean_only) then
+         where (.not.land_ice_mask) delta_t_surf = sst_new - t_surf
+         where (.not.land_ice_mask) t_surf = t_surf + delta_t_surf			 
 	 else
 	     delta_t_surf = sst_new - t_surf
 	     t_surf = t_surf + delta_t_surf
 	 endif
-	     
 end if
 
-if ((.not.do_sc_sst).or.(do_sc_sst.and.specify_sst_over_ocean_only)) then
+if(do_ape_sst) then 
```

Comment:
I think it would be nice to say what "ape" stands for (in my mind it's always available potential energy...)

Also, from a complete novice point of view, I wonder if it'd be good to have some notes about this SST choice somewhere at the level of the user interface / docs...

Reply:
I have added further description at L663 to say what APE stands for and the reference. Also at the point where the logic flag is declared. In terms of SST choices, @sit23 do you know if we have a standard location for this type of setting info?